stdlib/public/Concurrency/PartialAsyncTask.swift

```diff
@@ -18,3 +18,19 @@ public struct PartialAsyncTask {
 
   public func run() { }
 }
+
+
+public struct UnsafeContinuation<T> {
```

Comment:
This should be fixed-layout?

Reply:
Sure. Since `PartialAsyncTask` wasn't marked as fixed-layout either, I thought we were leaving annotations off of this for now. Do we build the module with library evolution on?